runtime/common/MeasureCounts.cpp

```diff
@@ -312,12 +313,16 @@ std::size_t sample_result::size(const std::string_view registerName) noexcept {
 }
 
 double sample_result::probability(std::string_view bitStr,
-                                  const std::string_view registerName) {
+                                  const std::string_view registerName) const {
   auto iter = sampleResults.find(registerName.data());
   if (iter == sampleResults.end())
     return 0.0;
 
-  return (double)iter->second.counts[bitStr.data()] / totalShots;
+  const auto countIter = iter->second.counts.find(bitStr.data());
+  if (countIter == iter->second.counts.end())
+    return 0.0;
+  else
```

Comment:
nit: else after return

Reply:
Good catch. I've updated the code to use a ternary operator instead.